CONTRIBUTING.md

```diff
@@ -22,12 +22,11 @@
 5. Create a PR for the release branch
 6. Once the release PR is landed to the `main` branch, checkout the `main` branch locally (remember to pull the remote changes) and run `npm run build`
 7. Run `git tag <VERSION_NUMBER>` with the new version and push the tag to git
-8. Ensure that the infrastructure repository is in the same directory as `ably-js` and is up to date with the remote main branch
-9. Run `npm run grunt -- release:deploy` (requires the SDKDeveloper AWS role) - pushes commit and tag, then publishes to the Ably CDN
-10. Run `npm publish .` (should require OTP) - publishes to NPM
-11. Visit https://github.com/ably/ably-js/tags and add release notes to the release (generally you can just copy the notes you added to the CHANGELOG)
-12. For nontrivial releases: update the ably-js submodule ref in the realtime repo
-13. Update the [Ably Changelog](https://changelog.ably.com/) (via [headwayapp](https://headwayapp.co/)) with these changes (again, you can just copy the notes you added to the CHANGELOG)
+8. Run `npm publish .` (should require OTP) - publishes to NPM
+9. Run the GitHub action "Publish to CDN" with the new tag name
```

Comment:
"with the new tag name" is incorrect. I would avoid being too specific here, as the `description` on the `version` input to the 'Publish to CDN' workflow already says enough. Perhaps take inspiration from how the 'Run the publish workflows' step is written [in `ably-asset-tracking-android`](https://github.com/ably/ably-asset-tracking-android/blob/main/CONTRIBUTING.md#release-process)?

Reply:
This was actually my suggestion and it's not immediately obvious to me why it's incorrect?

Reply:
ah, sorry... ok, so in `ably-asset-tracking-android` we have a `v` prefix on release tag names. Therefore, actually, this comment should have been targeting [the `description`](https://github.com/ably/ably-js/blob/1edb0d258f86f9313bfbf14e5e7d39ac6ec6da3b/.github/workflows/publish-cdn.yml#L6) as incorrect!

Perhaps that description should be 'Release tag', not 'Version, without v prefix'. 🤔 

(FWIW, I like the `v` prefix as it makes it a _bit_ more difficult for the publish script to checkout something that's not a release tag 🤷)

Reply:
Ah, that makes sense, I can see that we need to change that description 👍 

I like the `v` prefix too but it's a bit too late to change that for ably-js 😄

Reply:
I've updated the description of the gh action

Reply:
https://github.com/ably/ably-js/pull/889/commits/45484ca41c7b4e07bbe2bab6111ae81f25e87c38